subprojects/analysis-kotlin-descriptors/build.gradle.kts

```diff
@@ -8,6 +8,10 @@ plugins {
 }
 
 dependencies {
+    // to override some interfaces (JvmAnnotationEnumFieldValue, JvmAnnotationConstantValue) from compiler since thet are empty there
+    api(libs.intellij.java.psi.api) {
+        isTransitive = false
+    }
```

Comment:
Is it actually used anywhere or was there a problem related to it? Don't see any new code in the PR related to the mentioned classes. 

If it is actually needed, does it have to be `api` (i.e exposed to others) or can it be an `implementation`?